packages/wonder-blocks-modal/src/components/__docs__/one-pane-dialog.stories.js

```diff
@@ -3,15 +3,26 @@
 import * as React from "react";
 import {StyleSheet} from "aphrodite";
 
+import {
+    Breadcrumbs,
+    BreadcrumbsItem,
+} from "@khanacademy/wonder-blocks-breadcrumbs";
 import Button from "@khanacademy/wonder-blocks-button";
+import Color from "@khanacademy/wonder-blocks-color";
 import {View} from "@khanacademy/wonder-blocks-core";
 import {ActionMenu, ActionItem} from "@khanacademy/wonder-blocks-dropdown";
+import {Strut} from "@khanacademy/wonder-blocks-layout";
+import Link from "@khanacademy/wonder-blocks-link";
 import {ModalLauncher, OnePaneDialog} from "@khanacademy/wonder-blocks-modal";
-import {Body} from "@khanacademy/wonder-blocks-typography";
+import {Body, LabelLarge} from "@khanacademy/wonder-blocks-typography";
 
 import type {StoryComponentType} from "@storybook/react";
 
 import type {ModalElement} from "../../util/types.js";
```

Comment:
Hmm... so the `ModalElement` import was already here before I started working on this task. Something about this doesn't feel right. It's not a public type, so I don't think we should use it in the examples. Is there another way to type a modal? In the `WithLauncher` story, this is used where a modal is defined as follows:

```
const MyModal = ({closeModal}: MyModalProps): ModalElement => (...)
```

Reply:
@kevinbarabash Do you have any thoughts on this? Not sure if you're the right person to tag, but your name is on the last git blame.